contracts/LaunchList.sol

```diff
@@ -47,16 +47,17 @@ contract LaunchList is Ownable {
     // Uniswap V3 Oracle pool address
     address public uniswapV3OraclePool;
 
-    // Mapping of user addresses to their ETH spent during limited phases
-    mapping(address => uint256) public addressEthSpent;
+    // Mapping of user addresses to their USDC spent during limited phases
+    mapping(address => uint256) public addressUsdcSpent;
 
     // Purchase limits by phase
-    uint256 public phase1EthLimit = 1 ether;
-    uint256 public phase2EthLimit = 4 ether;
+    uint256 public phase1UsdcLimit = 1000 * 10 ** 6;
+    uint256 public phase2UsdcLimit = 4000 * 10 ** 6;
```

Comment:
Is this supposed to be `9000 * 10 ** 6` ?

Reply:
Yes, correct, I've updated it to be explicit - originally would be set on the day of the launch in line with prices, but I guess for USDC the defaults can carry through